### DISCUSSION TITLE
Well done

### DISCUSSION THREAD

**Discussion by AriTelias** [AxisCommunications/axis-model-zoo#25] [General]
Hi Mar, well done!
We can see that all the models seem to be 224 resolution. 
Do you recommend using 224 resolutions on real life, production applications?
Or will more models be added later?
Cheers
Ari

**Comment by Corallo** [comment]
Hello @AriTelias 

No, the 224x224 is just because we were showing mainly performance of Mobilenet, which happened to be available pretrained with that resolution. Also, Image classifiers are usually fine with lower resolution, unlike other type of networks. For other tasks, it might be good to have higher resolution if more precision is needed from the model.

In the future, we will expand the model zoo, showing performance of models with different resolutions.

**Comment by AriTelias** [comment]
Thanks Angelo, good job and have a great weekend!
